servers/audio_server.cpp

```diff
@@ -1487,7 +1487,10 @@ void AudioServer::init() {
 
 void AudioServer::update() {
 #ifdef DEBUG_ENABLED
-	if (EngineDebugger::is_profiling("servers")) {
+	// this prevents allocations every update, as "servers" does not exist
+	// 4.57 Mc    0.0%    1.40 Mc               AudioServer::update()
+	static StringName const SERVERS("servers", true);
+	if (EngineDebugger::is_profiling(SERVERS)) {
```

Comment:
I'm not sure what the correct fix is here, but this causes repeated allocations in the game loop.

Reply:
For this we usually allocate a static StringName. For example for the scene classes we use:
https://github.com/godotengine/godot/blob/2c136e6170a40f58f2dfb89d32eadfca7156ef37/scene/scene_string_names.cpp#L35

We also have a CoreStringNames class and an EditorStringNames class. Perhaps its time to create ServerStringNames?

Reply:
CC @KoBeWi

Reply:
Cool, I thought it might have been something like that, as I'd seen them elsewhere 👍🏻

Reply:
Actually should probably just go in core_string_names.cpp. That seems to be included everywhere

Reply:
You can just use SNAME.

Reply:
I can remove this change from the PR, if the other changes are interesting.

Reply:
Reverted this change from the PR

Reply:
Oh nice – I'll make that change then 👍🏻

Reply:
Done